temporalio/worker/worker.py

```diff
@@ -43,9 +45,12 @@ def __init__(
         *,
         task_queue: str,
         activities: Iterable[Callable] = [],
+        workflows: Iterable[Type] = [],
         activity_executor: Optional[concurrent.futures.Executor] = None,
+        workflow_task_executor: Optional[concurrent.futures.ThreadPoolExecutor] = None,
+        workflow_runner: WorkflowRunner = UnsandboxedWorkflowRunner(),
```

Comment:
I wouldn't expose this until we're sure we want to

Reply:
I am pretty sure I want though. I specifically wrote it for exposure because I know at least during research there will be many implementations. I expect `UnsandboxedWorkflowRunner` _not_ to be the default after research, but I still want it available for others to opt-out of sandboxing.

How about, after research and phase 3, if there is no value in exposing we can hide it? In the meantime, I'd really like sandboxing to be a configurable thing (even if we have no sandbox atm).

Reply:
Sure, we can revisit after phase 3. If we can sandbox and the price is reasonable I wouldn't expose the unsandboxed runner at all.

Reply:
> If we can sandbox and the price is reasonable I wouldn't expose the unsandboxed runner at all.

My fear is less about performance (though that does exist), and more that the sandbox will have caveats. Python is so dynamic that I may not be able to catch all strange uses or "I promise this is a harmless import side effect" cases. One example I could think of off the top of my head is if something is CPU heavy and therefore memoized but is essentially a cached, never changing value. Of course we might end up with a similar "sink" concept.

:+1: We'll see when we get there.